tools/wake-format/emitter.cpp

```diff
@@ -52,6 +56,14 @@
     return v;                                      \
   }
 
+#define MEMO_RESET()                         \
+  {                                          \
+    for (memo_map_t * map : __memo_maps__) { \
+      map->clear();                          \
+    }                                        \
+    __memo_maps__.clear();                   \
+  }
```

Comment:
I'm sure there's a good C++ reason , but this only seems to be used the once and feels like it doesn't need to be a `#define`.

Reply:
Me either, and if there's others following this pattern, it's probably good to match.

Reply:
SG!